src/routes/devices.ts

```diff
@@ -662,11 +662,13 @@ const upsertImageInstall = async (
 		}
 		await resinApi.patch({
 			resource: 'image_install',
+			id: {
+				installs__image: imageId,
+				device: deviceId,
```

Comment:
```suggestion
				device: deviceId,
				installs__image: imageId,
```
Reordered to match the sbvr :)